datalad_crawler/nodes/annex.py

```diff
@@ -857,7 +857,7 @@ def merge_branch(data):
                 elif strategy == 'theirs':
                     self.repo.merge(to_merge, options=["-s", "ours", "--no-commit"],
                                     expect_stderr=True, **merge_kwargs)
-                    self.repo._git_custom_command([], "git read-tree -m -u %s" % to_merge)
+                    self.repo.call_git(["git", "read-tree", "-m", "-u", to_merge])
```

Comment:
Almost missed this: Shouldn't `to_merge` extend the list rather than being included?

Reply:
Dunno about that yet, but `git` needs to be stripped from that list.

Reply:
`to_merge` is a single item from `all_to_merge` -- I think this is fine.